### ISSUE TITLE
Improve UI for Learn More updates

### ISSUE THREAD

**Issue by kainjow** [sparkle-project/Sparkle#1062]
If an update uses a `<link>` Sparkle will show a Learn More button. However, it also shows "Would you like to download it now?" and "Release Notes" above the web view.

Maybe these should be configurable? It doesn't really makes sense if one is trying to show a marketing message, especially for paid upgrades.

**Comment by pornel** [comment]
Right, the message could be improved. However, it's supposed to be localized, so it might be tricky to make it configurable.

**Comment by kainjow** [comment]
Maybe the appcast could be expanded for language support?

**Comment by pornel** [comment]
That would be relatively large effort for such a minor feature (hacking i18n in RSS, parsing it, changing internal feed representation, etc.)

OTOH I think I can simply change the UI string to cut off the "Would you like to download it now?" bit.

**Comment by kainjow** [comment]
> hacking i18n in RSS

but I thought RSS was XML and thus [extensible](https://github.com/sparkle-project/Sparkle/issues/427#issuecomment-56847515) ;)

Maybe for a future version, the update window UI could be generated on the fly, based on certain parameters, instead of hacking the nib. Should be fairly easy with auto layout.

**Comment by pornel** [comment]
OK, hack maybe is an exaggeration. There's `xml:lang` and we could use a made-up element.

But if you want to make a sales pitch with a custom layout, we already support [markup for this](https://html.spec.whatwg.org/multipage/).

**Comment by kainjow** [comment]
Yup html is already used for a pretty marketing message. Just think the other Sparkle text should be consistent if it's not an actual download.

**Comment by zorgiepoo** [comment]
I don't think this is still an issue.
